input/pagecontent/index.md

```diff
@@ -43,14 +43,14 @@ This guide is divided into several pages which are listed at the top of each pag
 - [Home](index.html): This page provides the introduction and scope for this guide.
 - [Conformance](conformance.html): This page describes the set of rules to claim conformance to this guide
 - [Guidance](guidance.html): These pages list the guidance for this guide. 
-  - [AU eRequesting Data for Interoperability](auereqdi.html): This page maps the AU eRequesting resources and elements to AUeReqDI data classes and data elements.
-  - [Relationship with other HL7 AU FHIR IGs](relationship.html): This page provides guidance on the relationship between AU eRequesting and other implementation guides.
+  - [AU eRequesting Data for Interoperability](auereqdi.html): This page maps AUeReqDI data groups and elements to FHIR artefacts in AU eRequesting.
+  - [Relationship with other IGs](relationship.html): This page provides guidance on the relationship between AU Core, AUCDI, and other implementation guides.
```

Comment:
Replace this line with eRequesting rather than Core

Reply:
Fixed, thanks.

Reply:
Still says AUCDI

Reply:
Fixed, sorry.